src/main/java/edu/harvard/iq/dataverse/api/Datasets.java

```diff
@@ -2191,7 +2191,7 @@ public Response returnToAuthor(@Context ContainerRequestContext crc, @PathParam(
             reasonForReturn = json.getString("reasonForReturn");
             // TODO: Once we add a box for the curator to type into, pass the reason for return to the ReturnDatasetToAuthorCommand and delete this check and call to setReturnReason on the API side.
             if (reasonForReturn == null || reasonForReturn.isEmpty()) {
```

Comment:
Hello @pdurbin and thank you @cmbz.
As a matter of fact, reason for return was already required in API endpoint and return `http error 400 Bad Request` if reason is null or empty.
And philosophically, I would say, what is the point of returning a submitted to review dataset if you don't add information on the reason ? Moreover, if it does not require an explanation, you can still type "No reason" or "Author just want to work on the draft" which reflects a form of reality if we are forced to enter a text. I don't see any issue with the required field. A global configuration would add a new entry to maintain, I would skip it ;)

Reply:
Oh! I had forgotten what the API behavior is. Thanks for the reminder! Now I fully support requiring a reason in the UI, for consistency with the API.